### PR TITLE
PB-6665: Cloud File missing Alert | Dashboard and Email Alerting Feature

### PR DESCRIPTION
**Pull request by t-nidhin**
alertmanager:
- added missing metadata ns
- Backup type message specific for failed and missing based on annotation.value

pxcentral-backup.yaml

- new rule added for cluster level permission to get services, which is required in torpedo to fetch px-backup installed ns

pxcentral-prometheus.yaml

- description added for backup missing
refer main reference: [px-backup](https://github.com/pure-px/px-backup/pull/1835)
refer ticket: [jira](https://purestorage.atlassian.net/browse/PB-6665)

### CODE REVIEW

**Review by ss-px — APPROVED** [review]

File: charts/px-central/templates/px-backup/pxcentral-prometheus.yaml [hunk 1]
```diff
@@ -522,17 +522,20 @@ spec:
...
-          {{ else }} part of backup schedule "{{ $labels.schedule_name}}" 
-          {{ end }} taken for cluster 
-          "{{ $labels.cluster}}" on namespace 
-          "{{ $labels.backup_namespace}}" has failed.`}}
-        summary: Backup failed
-      expr: pxbackup_backup_status == 4
-      for: 1m
-      labels:
-        severity: critical
+        - alert: BackupAlert
```
Comment (ss-px) [inline charts/px-central/templates/px-backup/pxcentral-prometheus.yaml:525]: I think these 2 lines have been repeated, please check:
```    
- alert: BackupAlert
      annotations:
```

File: charts/px-central/templates/px-backup/pxcentral-backup.yaml [hunk 2]
```diff
@@ -22,6 +22,9 @@ metadata:
     app.kubernetes.io/component: px-backup
 {{- include "px-central.labels" . | nindent 4 }}
 rules:
+  - apiGroups: [""]
```
Comment (px-kesavan) [inline charts/px-central/templates/px-backup/pxcentral-backup.yaml:25]: Why we require this service access in cluster scope ?

Comment (t-nidhin) [reply charts/px-central/templates/px-backup/pxcentral-backup.yaml:25]: we are now vendoring torpedo for auth functionality; and in init() px-backup installed namespace is found by searching all namespaces for the px-backup service. And where the px-backup service is found is marked as px-backup installed ns.
And torpedo requires cluster level service access

File: charts/px-central/templates/px-backup/pxcentral-prometheus.yaml (hunk 1 above)
Comment (t-nidhin) [reply charts/px-central/templates/px-backup/pxcentral-prometheus.yaml:525]: goo catch, Thanks, missed this in cherry pic

**Review by px-kesavan — APPROVED** [review]

**Pull request merged**